NASA-AMMOS/aerie-ui
# Add form for anchoring directives to other directives

This resolves #330.

It is also dependent on https://github.com/NASA-AMMOS/aerie/pull/570.

To test:
1. Create a plan with some activities
2. Select an activity
3. Check off the `Relative start time` checkbox in the `Selected Activity` form
4. Confirm that the anchor form is presented
5. Select the activity directive to anchor to
6. Input the offset
7. Confirm that the activity `Start Time` doy updates
